lms/static/js/ccx/schedule.js

```diff
@@ -139,7 +139,7 @@ var edx = edx || {};
               return !node.hidden;});
             this.$el.html(schedule_template({chapters: this.showing}));
             $('table.ccx-schedule .sequential,.vertical').hide();
-            $('table.ccx-schedule .toggle-collapse').on('click', this.toggle_collapse);
+            $('table.ccx-schedule .unit a').on('click', this.toggle_collapse);
```

Comment:
because link <a> has focus first (<a> are by default focusable) when user tries to expand or collapse tree, Now it is easy for screen readers to expand or collapse tree inside schedule tab ccx dashboard,